network_wrangler/transitnetwork.py

```diff
@@ -630,17 +634,16 @@ def _apply_individual_change(project_dictionary: dict):
         else:
             _apply_individual_change(project_card_dictionary)
 
-    def apply_python_calculation(
-        self, pycode: str, in_place: bool = True
-    ) -> Union(None, TransitNetwork):
+    def apply_python_calculation(self, pycode: str) -> "TransitNetwork":
         """
         Changes roadway network object by executing pycode.
 
         Args:
             pycode: python code which changes values in the roadway network object
-            in_place: update self or return a new roadway network object
         """
+        net = copy.deepcopy(self)
```

Comment:
Why do we need a deepcopy here? The same function in roadwaynetwork.py does not deepcopy.